apps/rds_tx.grc

```diff
@@ -1445,6 +1492,10 @@
       <value>14</value>
     </param>
     <param>
+      <key>radiotext</key>
+      <value>GNU Radio &lt;3</value>
+    </param>
+    <param>
       <key>ta</key>
```

Comment:
it is used in commit 4a5a22888a ... I admit that the PR is not as clean as it could be, it was a quick hack and I just wanted to share it.